int16[] ids
geometry_msgs/Pose2D[] poses
